web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -12,31 +12,45 @@ import { Certification, Funding, SidebarCardContent } from "@/lib/types/types";
 import analytics from "@/lib/utils/analytics";
 import { openInNewTab, scrollToTopOfElement, templateEval } from "@/lib/utils/helpers";
 import { isRemoteWorkerOrSellerBusiness } from "@businessnjgovnavigator/shared/domain-logic/businessPersonaHelpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
 import { Accordion, AccordionDetails, AccordionSummary, useMediaQuery } from "@mui/material";
 import { ReactElement, ReactNode, useRef, useState } from "react";
 
 export interface SidebarCardsListProps {
-  topCards: SidebarCardContent[];
-  bottomCards: SidebarCardContent[];
+  sideBarCards: SidebarCardContent[];
   fundings: Funding[];
   hiddenFundings: Funding[];
   certifications: Certification[];
   hiddenCertifications: Certification[];
-  displayFundings: boolean;
-  displayCertifications: boolean;
-  isCMSPreview?: boolean;
+  CMS_ONLY_DISPLAY_FUNDINGS?: boolean;
+  CMS_ONLY_DISPLAY_CERTIFICATIONS?: boolean;
+  CMS_ONLY_IS_REMOTE_SELLER_WORKER?: boolean;
 }
 
 export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement => {
   const [hiddenAccordionIsOpen, setHiddenAccordionIsOpen] = useState<boolean>(false);
   const { Config } = useConfig();
   const { business } = useUserData();
   const accordionRef = useRef(null);
+  const isDesktopAndUp = useMediaQuery(MediaQueries.desktopAndUp);
```

Comment:
[dust] Do we have a follow-up refactor ticket to remove this?

Reply:
https://www.pivotaltracker.com/story/show/187105184